DevelopedCalculator.js

```diff
@@ -0,0 +1,19 @@
+class DevelopedCalculator extends Calculator {
+    history = [];
+    render() {
+        super.render()
+            // first create a div 
+        const historyDiv = document.createElement("div");
+        // add the text to the div
+        historyDiv.appendChild(document.createTextNode("History:" + this.currentResult));
+        // prepend the historyDiv in the host element 
+        this.hostElem.prepend(historyDiv);
+    }
+
+    onButtonNumberClick_(event) {
```

Comment:
any update on the name of this method?

Reply:
then create onButtonNumberClick on the new class
it's very important that the name of the method matches with the method on the parent class

which name I have to choose more near to what I chosen ?